examples/SPDXJSONExample-v2.0.json

```diff
@@ -2,380 +2,267 @@
+      "filesAnalyzed" : "false",
+      "homepage" : "http://saxon.sourceforge.net/",
+      "licenseComments" : "Other versions available for a commercial license",
+      "licenseConcluded" : "MPL-1.0",
+      "licenseDeclared" : "MPL-1.0",
+      "name" : "Saxon",
+      "packageFileName" : "saxonB-8.8.zip",
+      "versionInfo" : "8.8"
+    }, {
+      "annotation" : [ {
+        "annotationDate" : "2011-01-29T18:30:22Z",
+        "annotationType" : "OTHER",
+        "annotator" : "Person: Package Commenter",
+        "comment" : "Package level annotation"
+      } ],
+      "checksum" : [ {
+        "algorithm" : "MD5",
+        "checksumValue" : "624c1abb3664f4b35547e7c73864ad24"
+      }, {
+        "algorithm" : "SHA1",
+        "checksumValue" : "85ed0817af83a24ad8da68c2b5094de69833983c"
+      }, {
+        "algorithm" : "SHA256",
+        "checksumValue" : "11b6d3ee554eedf79299905a98f9b9a04e498210b59f15094c916c91d150efcd"
+      } ],
+      "copyrightText" : "Copyright 2008-2010 John Smith",
+      "description" : "The GNU C Library defines functions that are specified by the ISO C standard, as well as additional features specific to POSIX and other derivatives of the Unix operating system, and extensions specific to GNU systems.",
+      "downloadLocation" : "http://ftp.gnu.org/gnu/glibc/glibc-ports-2.15.tar.gz",
+      "externalRef" : [ {
+        "referenceCategory" : "SECURITY",
+        "referenceLocator" : "cpe:2.3:a:pivotal_software:spring_framework:4.1.0:*:*:*:*:*:*:*",
+        "referenceType" : { }
+      }, {
+        "comment" : "This is the external ref for Acme",
+        "referenceCategory" : "OTHER",
+        "referenceLocator" : "acmecorp/acmenator/4.1.3-alpha",
+        "referenceType" : { }
+      } ],
+      "filesAnalyzed" : "true",
+      "hasFile" : [ "SPDXRef-JenaLib", "SPDXRef-DoapSource", "SPDXRef-CommonsLangSrc" ],
+      "homepage" : "http://ftp.gnu.org/gnu/glibc",
+      "licenseComments" : "The license for this project changed with the release of version x.y.  The version of the project included here post-dates the license change.",
+      "licenseConcluded" : "(LGPL-2.0-only OR LicenseRef-2)",
+      "licenseDeclared" : "(LGPL-2.0-only AND LicenseRef-2)",
+      "licenseInfoFromFiles" : [ "GPL-2.0-only", "LicenseRef-0", "LicenseRef-1" ],
+      "name" : "glibc",
+      "originator" : "Organization: ExampleCodeInspect (contact@example.com)",
+      "packageFileName" : "glibc-2.11.1.tar.gz",
+      "packageVerificationCode" : {
+        "packageVerificationCodeExcludedFile" : [ "excludes: ./package.spdx" ],
+        "packageVerificationCodeValue" : "d6a770ba38583ed4bb4525bd96e50461655d2758"
+      },
+      "sourceInfo" : "uses glibc-2_11-branch from git://sourceware.org/git/glibc.git.",
+      "summary" : "GNU C library.",
+      "supplier" : "Person: Jane Doe (jane.doe@example.com)",
+      "versionInfo" : "2.11.1"
+    } ],
+    "files" : [ {
```

Comment:
Same comment here as above, re: IDs -- I'm not seeing the SPDX ID definitions for the files?